website/source/docs/builders/lxc.html.md

```diff
@@ -0,0 +1,82 @@
+---
+description: |
+    The `lxc` Packer builder builds containers for lxc1. The builder starts an LXC
+    container, runs provisioners within this container, then exports the container
+    as a tar.gz of the root file system.
+layout: docs
+page_title: LXC Builder
+...
+
+# LXC Builder
+
+Type: `lxc`
+
+The `lxc` Packer builder builds containers for lxc1. The builder starts an LXC
```

Comment:
I see lxc1 in the docs but not referenced in the code anywhere. Is this only on Ubuntu? Does this code work on CentOS/RHEL or is Ubuntu required?